server/modules/record/service/recordService.js

```diff
@@ -90,16 +42,18 @@ const deleteRecord = async (user, surveyId, recordUuid) => {
   Log.debug('delete record. surveyId:', surveyId, 'recordUuid:', recordUuid)
 
   await RecordManager.deleteRecord(user, surveyId, recordUuid)
+  const userUuid = User.getUuid(user)
 
-  //notify users that record has been deleted
+  //notify the other users that record has been deleted
```

Comment:
i think this comment could be ```notify other users viewing or editing the record that it has been deleted```. what do you think?

Reply:
Ok better